pyanaconda/ui/gui/spokes/welcome.py

```diff
@@ -112,9 +113,10 @@ def apply(self):
     @property
     def completed(self):
         # Skip the welcome screen if we are in single language mode
+        # or module is not available.
         # If language has not been set the default language (en_US)
         # will be used for the installation and for the installed system.
-        if flags.flags.singlelang:
+        if flags.flags.singlelang or not is_module_available(LOCALIZATION):
```

Comment:
It would be nice to use `should_run` as well if possible, but this solution is good enough for now.